### ISSUE TITLE
PI: Aborting a PI and transferring the results transfers the last "current" parameter values

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#391]
When a PI run is aborted, not the "best" but the "current" values are transferred to the simulation. This is clearly wrong, only the best results should be applied.

**Comment by rwmcintosh** [comment]
I can't reproduce this issue. I ran a test on a single PI and I can see that during the run of the PI the 'Best' is always updated after a single run (if it's error is better) and that 'Best' is always used to transfer.

This was entered some time ago, so it could have been fixed somehow. Although, I'm looking at the annotations in Git and I don't see it clearly.

**Comment by PavelBal** [comment]
Can you somehow abort it after a run where the values are worse than the best?

**Comment by rwmcintosh** [comment]
Yeah, I stopped after the first run and 'best' was updated. After second run 'best' was not updated because the error was higher than the current 'best'.

**Comment by PavelBal** [comment]
Seems to be fixed then.
